client/login/wp-login/login-links.jsx

```diff
@@ -84,7 +85,8 @@ export class LoginLinks extends React.Component {
 				icon={ true }
 				onClick={ this.recordHelpLinkClick }
 				target="_blank"
-				href="https://en.support.wordpress.com/security/two-step-authentication/">
+				href="https://en.support.wordpress.com/security/two-step-authentication/"
```

Comment:
That was just to be consistent with the convention adopted in the rest of this file.